libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -812,6 +812,13 @@
         "tabLabel": "Udgående"
       }
     },
+    "status": {
+      "Received": "Modtaget",
+      "AwaitingDispatch": "Afventer afsendelse",
+      "AwaitingReply": "Afventer svar",
+      "Accepted": "Accepteret",
+      "Rejected": "Afvist"
+    },
```

Comment:
nitpick: let's reuse the translations under 'outgoingMessages.shared.documentStatus'